distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1518,6 +1530,79 @@ public void testRetrievalReliabilityFails() throws Exception {
                 "FAILED");
     }
 
+    @Test
+    public void testMetacardCache() throws Exception {
+
+        //Start with a clean cache
+        console.runCommand(CLEAR_CACHE);
+
+        long start = System.currentTimeMillis();
+        long waitTime = 20000;
+        boolean resultsFound = true;
+        while (resultsFound && System.currentTimeMillis() - start < waitTime) {
+            resultsFound = !console.runCommand(QUERY_CACHE)
+                    .contains(" 0 result(s)");
```

Comment:
I think it would be less brittle if we injected the catalog framework and did a query on the cache that way instead of parsing the output of the cache command.